targets/TARGET_Realtek/TARGET_AMEBA/sdk/common/drivers/wlan/realtek/rtw_emac.cpp

```diff
@@ -0,0 +1,244 @@
+/* mbed Microcontroller Library
```

Comment:
Here as well, it would be good to move it to higher level directory, as it's mbed OS API implementation.